Tidy examples

sftim opened the pull request:
- fix a spelling mistake
- ~also use `eprintln!` in place of `println!`~
- switch to paste tense for actions that already occurred

Commit:
Fix spelling in comment

rillian reviewed (commented):
Spelling fix is correct.

As for `eprintln!` I don't see why you'd change this message, but not all the others, which are similarly diagnostic?

There are also `ringbahn::(e)println!` wrappers, but I don't understand what the distinction is or whether they'd be more appropriate here.

sftim commented:
I didn't see any other examples using `println!`. I can look again.

If I change non-example code, I'll do that in a separate PR.

rillian commented:
The more-recent  `read-event` example [uses](https://github.com/ringbahn/ringbahn/blob/7cb9499/examples/read-event.rs#L20) `ringbahn::print!` which afaict is an async version of the macro. So possible in this context, but perhaps not necessary? I guess for this particular line it would allow the server to accept new connections if logging output blocked, but that might just get you OOM killed instead.

As for changing only one message, I meant that the second patch does `s/println/eprintln/` on the "connection received" message, but not on the "listening", "read bytes", or "write bytes" messages in the same example. If one goes to the stderr, shouldn't they all? Although, I think it's more usual to leave log messages like this on stdout unless they would corrupt some other information stream; can be slightly less work to redirect to a log file. So I would just leave it as `println!`.

Commit:
Change tense of debug output

Write verbs in past tense to make it clear these have already happened
(these async calls have been awaited).

sftim commented:
Revised, thanks for the feedback.

rillian reviewed (approved):
LGTM!

(nb not a maintainer, just trying to help out.)
